packages/webos/LS2Request/LS2Request.js

```diff
@@ -55,10 +66,22 @@ export default class LS2Request {
 		this.bridge.onservicecallback = this.callback.bind(this, onSuccess, onFailure, onComplete);
 		this.bridge.call(adjustPath(service) + method, JSON.stringify(parameters));
 
+		// timer will be cleared when the callback is run, so this should only fire if the timeout threshold is reached
```

Comment:
Could `this.bridge.call` be synchronous for any calls?  If so, timer will be clear when `callback` is called and the timer will be set up after.  Would it be safer to reorder these lines?